src/llama-context.cpp

```diff
@@ -1275,20 +1278,12 @@ uint32_t llama_context::output_reserve(int32_t n_outputs) {
 // graph
 //
 
-int32_t llama_context::graph_max_nodes() const {
-    return std::max<int32_t>(65536, 5*model.n_tensors());
+uint32_t llama_context::graph_max_nodes() const {
+    return std::max<uint32_t>(65536u, 5u*model.n_tensors());
```

Comment:
On a side note, I don't know why the minimum number of nodes is so big. This is a waste of memory for the smaller models. The change seems to be from #11571.

Reply:
I will follow-up with a PR to fix this.